lib/ansible/module_utils/redhat.py

```diff
@@ -69,9 +69,9 @@ def update_plugin_conf(self, plugin, enabled=True):
             cfg.read([tmpfile])
 
             if enabled:
-                cfg.set('main', 'enabled', 1)
+                cfg.set('main', 'enabled', '1')
```

Comment:
Could you please add a changelog entry like - [here](https://github.com/ansible/ansible/tree/devel/changelogs/) ?

```
bugfixes:
- rh_register - For compatibility using the rh_register module on hosts set to use a python 3 interpreter, use string values (https://github.com/ansible/ansible/issues/65252).
```